nextflow/modules/talos/HPOFlagging/main.nf

```diff
@@ -13,19 +13,15 @@ process HPOFlagging {
         path talos_config
 
     output:
-        path "${params.cohort}_pheno_annotated_report.json", emit: "pheno_annotated"
-        path "${params.cohort}_pheno_filtered_report.json", emit: "pheno_filtered"
+        path "${params.cohort}_pheno_annotated_report.json"
```

Comment:
dropping a second output here, there's no need for 2 reports in the new framework